hfaerber/Palette_Picker
# Login component

### What does this PR do? 

- Creates and styles the login component. 
- Adds a testing file for the component with some basic tests.

----------

### How should this be manually tested? 

- `git fetch`
- `git checkout login-component`
- `npm install` (I added a serializer, but I'm not sure that it's needed)
- 'npm start` to see the page in the browser. Try clicking the button with one or more inputs empty to ensure you get the error. Check the state of Login to make sure that the state hooks are working onChange of both inputs. After filling both out, click the button and App should now have the user in it's state, you should also be redirected to `/home`.
- 'npm test` to run tests. (Second test is failing)

----------

### Any background context you'd like to provide? 

- The second test is failing because it cannot find any nodes with the selected id. If you look at the snapshot it just appears as a blank object which makes me think that something is wrong with the rendering entirely. I have tried recreating the snapshot and selecting different nodes but have had the same result. 

----------

### Screenshots: 

<details>

<summary>Login in browser</summary>

 ![ ](https://user-images.githubusercontent.com/47998896/73894992-5edc7d00-483b-11ea-80d6-10dd855e22df.png)

</details>

<details>

<summary>Login in browser w/ error</summary>

 ![ ](https://user-images.githubusercontent.com/47998896/73895043-83d0f000-483b-11ea-94d9-165e418d301e.png)


</details>
